Author: <doko@ubuntu.com>
Description: In some cases, code that is intended to be used in the presence of multibyte 
characters is called when no such characters are present, leading to incorrect 
display position calculations and incorrect redisplay. 
--- a/display.c
+++ b/display.c
@@ -2380,6 +2380,8 @@
 
   if (end <= start)
     return 0;
+  if (MB_CUR_MAX == 1 || rl_byte_oriented)
+    return (end - start);
 
   memset (&ps, 0, sizeof (mbstate_t));
 
